app/views/workload-points.html

```diff
@@ -16,9 +16,7 @@
 
 <p>Workload points are set here and are used to calculate capacity points for all Offender Managers throughout the Workload Management Tool.</p>
 
-<!-- TODO: Include user. -->
-
-<p>Current Workload Points created by <span class="bold">test user</span> on <span class="bold">{{ wp.effectiveFrom }}</span></p> 
+<p>Current Workload Points created by <span class="bold">{{ wp.updatedByUserId }}</span> on <span class="bold">{{ wp.effectiveFrom }}</span></p> 
```

Comment:
As mentioned above, this should be a name by the time it's rendered so we could update the property name.

Reply:
Addressed above